docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,281 @@
+              schema:
+                $ref: '#/components/schemas/VerifiableCredential'
+        default:
+          $ref: '../common/error_response.yaml'
+
+  /internal/vcr/v2/issuer/vc/search:
+    get:
+      summary: "Resolves a verifiable credential issued by this node"
+      description: >
+        The resolutionResult contains the credential regardless of the validity.
+        The resolutionResult.currentStatus must be checked to see if the credential is valid.
+
+        error returns:
+        * 404 - Corresponding credential could not be found
+        * 500 - An error occurred while processing the request
+      operationId: "resolveIssuedVC"
+      parameters:
+        - name: credentialType
+          in: query
+          description: The type of the credential
+          example: NutsOrganizationCredential
+          required: true
+          schema:
+            type: string
+        - name: issuer
+          in: query
+          description: the did of the issuer
+          example: did:nuts:123
+          required: true
+          schema:
+            type: string
+        - name: subject
+          in: query
+          description: the did of the issuer
+          example: did:nuts:456
+          required: false
+          schema:
+            type: string
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Credential has been found and is returned.
+          content:
+            application/vc+json:
+              schema:
+                $ref: '#/components/schemas/ResolutionResult'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/issuer/vc/{id}:
+    parameters:
+      - name: id
+        in: path
+        description: URL encoded ID.
+        required: true
+        example:
+          - "did:nuts:B8PUHs2AUHbFF1xLLK4eZjgErEcMXHxs68FteY7NDtCY#90382475609238467"
+        schema:
+          type: string
+    delete:
+      summary: "Revoke an issued credential"
+      description: |
+        Revoke a credential.
+
+        error returns:
+        * 400 - Credential can't be revoked. Most likely due to a missing private key.
+        * 404 - Credential is not found
+        * 409 - Credential has already been revoked
+        * 500 - An error occurred while processing the request
+      operationId: "revokeVC"
+      tags:
+        - credential
+      responses:
+        "200":
+          description: Revocation has been processed locally. It has also been published to the network.
+          content:
+            application/vc+json;type=revocation:
+              schema:
+                $ref: '#/components/schemas/Revocation'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vcr/v2/verifier/vc:
+    post:
+      summary: Verifies a Verifiable Credential
+      description: |
+        Verifies a Verifiable Credential: It checks the signature, expiration, revocation status and if the issuer is trusted.
```

Comment:
And validity of the issuer at the time of issuance?

Reply:
What makes an issuer valid? Should we check for deactivation?